python/tvm/contrib/pyfrontend.py

```diff
@@ -0,0 +1,305 @@
+"""Compiling a subset of Python to HalideIR"""
+import ast
```

Comment:
Good to learn: Ast is a python package, which is similar with Ast in halide. You may add a comment for code readers.

Reply:
Actually, this is not the cardinal problem. The cardinal problem I am suffering from is that `ast` in py2 and py3 are quite different which makes me not pass the test yet.